src/Razor/test/RazorLanguage.Test/IntegrationTests/ComponentCodeGenerationTestBase.cs

```diff
@@ -4281,7 +4281,7 @@ public void EventHandlerTagHelper_EscapeQuotes()
         {
             // Act
             var generated = CompileToCSharp(@"
-<input @onfocus='alert(""Test"");' />
+<input onfocus='alert(""Test"");' />
```

Comment:
Same question.

Reply:
Ditto

Reply:
It's still useful as a test, but its meaning has changed. It's a JS event handler. We should have some tests for this because it needs to be possible, but it's not different from any other attribute.